### ISSUE TITLE
Indexers don't show in the backend.

### ISSUE THREAD

**Issue by danslo** [danslo/hhvm#3]
Very likely a SimpleXML bug.

``` php
$indexes = Mage::getConfig()->getNode(Mage_Index_Model_Process::XML_PATH_INDEXER_DATA); 
```

Should get indexers but doesn't. Need to create a pure PHP test-case for this.

**Comment by danslo** [comment]
This is about as close as I can get hiphop to fail:

``` php
<?php

require_once 'app/Mage.php';

Mage::init();

$indexes = Mage::getConfig()->getNode('global/index/indexer');
$children = $indexes->children();
var_dump($children);
```

If anyone is up for turning the above testcase into pure PHP testcase, please do so.

**Comment by danslo** [comment]
Fixed with 7216b04c25bfed14a62e52e0b760afc51aabd079.
